docs/apache-airflow/concepts/priority-weight.rst

```diff
@@ -0,0 +1,32 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _concepts:priority-weight:
+
+Priority Weights
+================
+
+``priority_weight`` defines priorities in the executor queue. The default ``priority_weight`` is ``1``, and can be
+bumped to any number. When sorting the executor queue to evaluate which task should be executed next, we calculate the
+true ``priority_weight``, which is summed up with all of the ``priority_weight`` values from its downstream tasks.
```

Comment:
This is true only if `weight_rule='downstream'`. I think we should explain also about weight_rule parameter in this section.

Reply:
Good idea! I could only find documentation mentioning weight_rule from the [BaseOperator docstring](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/models/index.html?highlight=weight_rule#airflow.models.BaseOperator).